.ci/install_crio.sh

```diff
@@ -55,23 +55,20 @@ then
 		else
 			crio_version=$(get_version "externals.crio.meta.openshift")
 		fi
+		crictl_version=$(get_test_version "externals.critools.openShift3Version")
+		crictl_url=https://github.com/kubernetes-sigs/cri-tools/releases/download/v$crictl_version/crictl-$crictl_version-linux-"$(${cidir}/kata-arch.sh -g)".tar.gz
 	else
 		crio_version=$(get_version "externals.crio.version")
+		crictl_version=$(get_test_version "externals.critools.version")
+		crictl_url=https://github.com/kubernetes-sigs/cri-tools/releases/download/$crictl_version/crictl-$crictl_version-linux-"$(${cidir}/kata-arch.sh -g)".tar.gz
 	fi
 
 	git fetch
 	git checkout "${crio_version}"
 fi
 
-echo "Get CRI Tools"
-critools_repo="${kubernetes_sigs_org}/cri-tools"
-go get "$critools_repo" || true
-pushd "${GOPATH}/src/${critools_repo}"
-critools_version=$(grep "ENV CRICTL_COMMIT" "${GOPATH}/src/${crio_repo}/Dockerfile" | cut -d " " -f3)
-git checkout "${critools_version}"
-go install ./cmd/crictl
-sudo install "${GOPATH}/bin/crictl" /usr/bin
-popd
+echo "Installing CRI Tools"
+wget -qO- "$crictl_url" | tar xfz - -C /usr/bin
```

Comment:
- I see you are updating already existing code but I think we need a check in here to test for [`KATA_DEV_MODE`](https://github.com/kata-containers/tests#developer-mode) as devs probably don't want their `/usr/bin/` stamped on (but `/usr/local/` would be "fair game" I think).

- We avoid `wget` where possible so please could you convert this to use `curl`.

Reply:
I think we could generally use `/usr/local`, then an additional check would not be needed right?
I changed the command to use `curl` instead of `wget`